src/libServer/Server.cpp

```diff
@@ -102,11 +102,11 @@ Json::Value Server::CreateTransaction(const Json::Value& _json) {
 
     const PubKey& senderPubKey = tx.GetSenderPubKey();
     const Address fromAddr = Account::GetAddressFromPublicKey(senderPubKey);
+    const Account* sender = AccountStore::GetInstance().GetAccount(fromAddr);
```

Comment:
Should check whether `sender == nullptr` here

Reply:
good point.